pixlie_ai/src/entity/web/scraper.rs

```diff
@@ -567,17 +494,17 @@ pub fn scrape(node: &NodeItem, engine: Arc<&Engine>) -> PiResult<()> {
 
     match &node.payload {
         Payload::Text(payload) => {
-            if payload.contains("%PDF-") || payload.contains("trailer") || payload.contains("%%EOF")
-            {
+            if !payload.to_lowercase().contains("<html") && !payload.contains("<!DOCTYPE html") {
```

Comment:
It's common for webpages to miss the doctype in the beginning. I don't think it's a good idea to include that in the check.
I'd rather store extension/mimetype in the fetcher and use that as a check here. Though, that can be a part of a separate PR.

Reply:
Not surprised. Hacker News website does not have `<!DOCTYPE>` declaration.
Perhaps a better idea to remove that check.